src/ostree/ot-builtin-refs.c

```diff
@@ -60,7 +70,60 @@ static gboolean do_ref (OstreeRepo *repo, const char *refspec_prefix, GCancellab
           g_print ("%s\n", ref);
         }
     }
+  else if(opt_create)
+    {
+      gboolean old_ref_exists = FALSE;
+      const char *checksum = NULL;
+
+      g_hash_table_iter_init (&hashiter, refs);
+      while (g_hash_table_iter_next (&hashiter, &hashkey, &hashvalue))
```

Comment:
Delete needs to match *multiple* things so iterating over all of them makes sense, whereas here we only need to look at exactly two, right?